### ISSUE TITLE
Ближайшие соседи

### ISSUE THREAD

**Issue by Bocha087** [Bocha087/Prediction#1]
import pandas as pd
from sklearn.neighbors import KNeighborsClassifier
from sklearn.model_selection import train_test_split
import joblib

# загрузка данных
data = pd.read_csv('diabetes2.csv')

# разделение данных на признаки и целевой признак
X = data.iloc[:, :-1]
y = data.iloc[:, -1]

# разделение данных на обучающую и тестовую выборки
X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.2, random_state=42)

# создание модели и обучение на обучающей выборке
model = KNeighborsClassifier(n_neighbors=5)
model.fit(X_train, y_train)

# оценка точности модели на тестовой выборке
accuracy = model.score(X_test, y_test)
print("Accuracy: ", accuracy)

# сохранение модели в файл
joblib.dump(model, 'knn_model.joblib')

# загрузка данных
data = pd.read_csv('diabtest.csv')

# разделение данных на признаки и целевой признак
X = data.iloc[:, :-1]

# загрузка обученной модели
model = joblib.load('knn_model.joblib')

# предсказание значения целевого признака
prediction = model.predict(X)

# вывод результата
print("Predicted diabetes probability:", prediction)

**Comment by Bocha087** [comment]
[diabetes.csv](https://github.com/Bocha087/Prediction/files/11368985/diabetes.csv)
